tokio-trace-futures/examples/proxy_server.rs

```diff
@@ -99,7 +99,7 @@ fn main() -> Result<(), Box<std::error::Error>> {
                 })
                 .instrument(span!(
                     "transfer completed",
-                    client_address = field::debug(&client_addr),
+                    client_address = field::debug(&client_addr.unwrap()),
```

Comment:
Using `unwrap()` here will cause the program to panic if the client address is `None`. I would just print the `Option` instead.